### PR TITLE
[NFC] Add in drupal 8  info.yml file for civicrm_webtest module

### PR DESCRIPTION
**Pull request by seamuslee001**

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
@colemanw @totten is this module still used?

File: tools/drupal/modules/civicrm_webtest/civicrm_webtest.info.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: CiviCRM WebTest Fixtures
+description:  Configuration data for CiviCRM WebTest
+version: 4.3
+dependencies:
+  - civicrm:civicrm
+package: CiviCRM
+type: module
+core: 8.x
+php = 5.3
```
Comment (eileenmcnaughton) [inline tools/drupal/modules/civicrm_webtest/civicrm_webtest.info.yml]: php 7.0

**Comment by totten** [comment]
It is still used for setting up the `demo` permission. There's a parallel issue around Backdrop: https://lab.civicrm.org/infra/ops/issues/906#note_21974

My general sense is that the list of demo user permissions should be moved to a JSON or TXT file  and used by all manner of demo builds (D7/D8/BD/WP).

**Comment by seamuslee001** [comment]
@totten would you expect that to be in buildkit that list or in core?

**Comment by totten** [comment]
It kinda works either way, doesn't it?

The information feels more akin to content under `civicrm-buildkit.git/app/config/*/install.sh` (i.e. list of things to setup during installation). OTOH, if you think about how `civicrm-core` goes through releases and how the list of available permissions is liable to evolve in tandem with `civicrm-core`, then it feels more convenient under `civicrm-core.git/tools/...`.

I'd say dealer's choice.

**Pull request closed without merging**